package.json

```diff
@@ -13,9 +13,12 @@
     "lint": "eslint './**/*.{js,ts,tsx}'"
   },
   "dependencies": {
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
     "@headlessui/react": "1.6.6",
     "@iconscout/react-unicons": "^1.1.6",
     "@internxt/lib": "1.2.0",
+    "@mui/material": "^5.10.11",
```

Comment:
why is `@mui/material` required? This library is big

Reply:
Because I use Snackbar component from this library.

Reply:
Maybe I need search other package only with Snackbar component?